src/combinatorics/simple_graph/basic.lean

```diff
@@ -1039,11 +1071,27 @@ map_adj_iff f
   end }
 
 /-- Embeddings of types induce embeddings of complete graphs on those types. -/
-def complete_graph.of_embedding {α β : Type*} (f : α ↪ β) : complete_graph α ↪g complete_graph β :=
+def complete_graph.of_embedding {α β : Type*} (f : α ↪ β) :
+  (⊤ : simple_graph α) ↪g (⊤ : simple_graph β) :=
```

Comment:
Do you want to deprecate `complete_graph`? If so, that should probably be a separate PR.

Unrelatingly, `simple_graph.embedding.complete_graph.of_embedding` is a ludicrous (accidental?) name. I would like it to be `simple_graph.embedding.complete_graph`.

Reply:
There was a discussion a while back about preferring `⊤` over `complete_graph`. The docstring for `complete_graph` is meant to indicate this, but [simple_graph.complete_graph_eq_top](https://leanprover-community.github.io/mathlib_docs/combinatorics/simple_graph/basic.html#simple_graph.complete_graph_eq_top) makes the stance clear. The point in having `complete_graph` as its own definition is to help people find how to create a complete graph.

I agree the name is silly, but I didn't want to change it in this PR. The reason this was modified was that in some version I think I was using it, but now it's not. I'll just split it off into its own PR.